x-pack/plugins/file_data_visualizer/public/application/components/results_links/results_links.tsx

```diff
@@ -33,6 +26,17 @@ interface Props {
   showFilebeatFlyout(): void;
 }
 
+interface GlobalState {
+  time?: {
+    from: string;
+    to: string;
+    mode?: 'absolute' | 'relative';
+  };
+  refreshInterval?: { pause: boolean; value: number };
```

Comment:
How about using `RefreshInterval` type, from src/plugins/data/public, here instead of redefining it?